fluss-lake/fluss-lake-paimon/src/test/java/com/alibaba/fluss/lake/paimon/flink/FlinkUnionReadPrimaryKeyTableITCase.java

```diff
@@ -202,6 +206,58 @@ void testPrimaryKeyTable(boolean isPartitioned) throws Exception {
         assertThat(rows.toString()).isEqualTo(sortedRows(expectedRows).toString());
     }
 
+    @Test
+    void testUnionReadWithTimeStamp() throws Exception {
```

Comment:
I created https://github.com/alibaba/fluss/issues/1127 used to improve our test coverage in next version